src/EnergyPlus/UnitarySystem.cc

```diff
@@ -4235,19 +4168,19 @@ namespace UnitarySystems {
             } else {
                 Fans::GetFanType(state, loc_m_FanName, this->m_FanType_Num, isNotOK, cCurrentModuleObject, loc_m_FanName);
                 if (isNotOK) {
-                    ShowContinueError(state, "Occurs in " + cCurrentModuleObject + " = " + thisObjectName);
+                    ShowContinueError(state, format("Occurs in {} = {}", cCurrentModuleObject, thisObjectName));
                     errorsFound = true;
                 } else {
                     isNotOK = false;
                     ValidateComponent(state, loc_fanType, loc_m_FanName, isNotOK, cCurrentModuleObject);
                     if (isNotOK) {
-                        ShowContinueError(state, "Occurs in " + cCurrentModuleObject + " = " + thisObjectName);
+                        ShowContinueError(state, format("Occurs in {} = {}", cCurrentModuleObject, thisObjectName));
                         errorsFound = true;
                     } else { // mine data from fan object
                         // Get the fan index
                         Fans::GetFanIndex(state, loc_m_FanName, this->m_FanIndex, errFlag);
```

Comment:
I'm sure you already know this (at least implicitly), but once you've gotten the FanIndex here you don't need to keep searching for it again and again in `GetFanDesignVolumeFlowRate`, `GetFanInletNode`, `GetFanOutletNode`, etc. You can just use declare `auto &fan = state.dataFans->Fans(FanIndex);` and just pick off the fields you need from that.

Reply:
Lot's of this completed, more to go.

Reply:
Most of the indexing is completed. I wouldn't doubt if there is a few more places for this type of change.